integration-tests/src/test/java/org/apache/fineract/integrationtests/ExternalBusinessEventTest.java

```diff
@@ -942,8 +942,8 @@ public void testInterestPaymentWaiverNotReverseReplayOnCreationAndHasGeneratedEx
             externalEventHelper.enableBusinessEvent("LoanAdjustTransactionBusinessEvent");
             AtomicReference<Long> loanIdRef = new AtomicReference<>();
             runAt("15 January 2025", () -> {
-                PostLoanProductsResponse loanProductResponse = loanProductHelper
-                        .createLoanProduct(create4IProgressive().isInterestRecalculationEnabled(true));
+                PostLoanProductsResponse loanProductResponse = loanProductHelper.createLoanProduct(create4IProgressive()
+                        .isInterestRecalculationEnabled(true).recalculationRestFrequencyType(2).recalculationRestFrequencyInterval(1));
```

Comment:
I think we have these enums in the BaseLoanIntegrationTest class (at the end), don't we?